src/sql/gazetteer_lol_extract_view.sql

```diff
@@ -9,7 +9,9 @@
 --
 -- ################################################################################
 
-﻿CREATE OR REPLACE VIEW gazetteer.name_export_for_lol AS
+set search_path= gazetteer, public;
```

Comment:
It's generally better to avoid changing sessions configuration in these sql files.
A function can have a search path associated with it, adding the `SET search_path` statement in the `CREATE FUNCTION` statement.

Reply:
Thanks @strk that is good to know. 

The entire project is set up like this so I will leave it for now